### PR TITLE
Try to fix erros raise during server install under ubuntu.

### PR DESCRIPTION
**Pull request by VertexC**
Basically add OS check and fix the import issue via export PYTHONPATH.

### CODE REVIEW

File: install/install.sh [hunk 1]
```diff
@@ -18,15 +18,19 @@ echo ""
 echo "* Updating package list        "
 apt-get update >> "$install_log" 2>&1
 echo "* Installing nginx, python & pip      "
-apt-get -q -y install dnsutils nginx python python-dev python-pip >> "$install_log" 2>&1
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
```
Comment (canihavesomecoffee) [inline install/install.sh:22]: I'd rather see a statement that installs the common dependencies, and then a small section to just install platform specific.

I also wonder why it's necessary to install those, while they're seemingly present by default on an rpi...

File: install/install.sh [hunk 2]
```diff
@@ -18,15 +18,19 @@ echo ""
...
 if [ ! -f /etc/init.d/mysql* ]; then
     echo "* Installing MySQL (root password will be empty!)"
     DEBIAN_FRONTEND=noninteractive apt-get install -y mysql-server >> "$install_log" 2>&1
 fi
 echo "* Update setuptools            "
-easy_install -U setuptools >> "$install_log" 2>&1
+pip install --upgrade setuptools  >> "$install_log" 2>&1
 echo "* Installing pip dependencies"
-pip install twisted pyopenssl flask-sqlalchemy flask passlib pymysql service_identity pycrypto flask-wtf netifaces gunicorn >> "$install_log" 2>&1
+pip install werkzeug ipaddress enum34 cryptography idna sqlalchemy twisted pyopenssl flask-sqlalchemy flask passlib pymysql service_identity pycrypto flask-wtf netifaces gunicorn >> "$install_log" 2>&1
```
Comment (canihavesomecoffee) [inline install/install.sh]: werkzeug should be inferred by installing flask... Is this no longer the case?

File: bin/pipotd [hunk 3]
```diff
@@ -13,6 +13,7 @@ function usage() {
     echo -e "\t\t--help\tThis help\n"
 }
 
+export PYTHONPATH=/usr/src/pipot/server/
```
Comment (canihavesomecoffee) [inline bin/pipotd]: a) This is hardcoded (not good)
b) Is this really necessary? If launched properly it shouldn't be necessary I think.

**Review by canihavesomecoffee — CHANGES_REQUESTED** [review]

File: install/install.sh (hunk 1 above)
Comment (VertexC) [reply install/install.sh:22]: Fixed that in the new commit.

Ummm, I haven't got my rpi yet. But it seems that they are needed under ubuntu.

File: install/install.sh (hunk 2 above)
Comment (VertexC) [reply install/install.sh]: remove werkzeug in the latest commit.

File: bin/pipotd (hunk 3 above)
Comment (VertexC) [reply bin/pipotd]: change it by using $DIR

I think it is reasonable for it to have import error. As the pwd here is server/bin, while config_parser and  others are under server/.

File: install/install.sh (hunk 2 above)
Comment (canihavesomecoffee) [reply install/install.sh]: It's still in here :)

**Review by canihavesomecoffee — CHANGES_REQUESTED** [review]

**Review by canihavesomecoffee — APPROVED** [review]

**Pull request merged**